source/agora/common/BlockStorage.d

```diff
@@ -33,17 +33,8 @@ import std.path;
 import std.stdio;
 
 
-/// The size of header
-private immutable size_t MFILE_HEAD_SIZE = size_t.sizeof;
-
-/// Amount of reserved size
-private immutable size_t MFILE_RESERVE_SIZE = 64 * 1024;
-
-/// The maximum number of block in one file
-private immutable ulong MFILE_MAX_BLOCK = 100;
```

Comment:
In this commit (the second) those constants are removed, not renamed, so this commit does not compile.

Reply:
fixed

Reply:
fixed